bedrock/mozorg/forms.py

```diff
@@ -51,7 +51,7 @@ def render(self, name, value, attrs=None):
         input_txt = super(PrivacyWidget, self).render(name, value, attrs)
 
         policy_txt = _(u'I’m okay with Mozilla handling my info as explained '
-                       u'in <a href="%s">this Privacy Notice</a>')
+                       u'in <a href="%s" style="color:#000">this Privacy Notice</a>')
```

Comment:
Thank you @alexgibson, I will make these changes later today, I appreciate the help!